cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/rolling/AdminClient.java

```diff
@@ -17,12 +16,30 @@
 import java.util.Map;
 
 /**
- * An abstraction over both a Kafka Admin client, and a Kafka Agent client.
+ * An abstraction over a Kafka Admin client.
  */
-interface RollClient {
+interface AdminClient {
```

Comment:
Did you mean to rename this? I think it's confusing if Kafka has an `Admin` client and we have an `AdminClient`.

Reply:
I have reverted back this renaming. We now have a separate abstract class for KafkaAgent, which is called AgentClient but I think this might be also confusing because we have `KafkaAgentClient` already.